study.md

```diff
@@ -19,38 +19,40 @@ Please write the code that would take the following integer `007` and return the
 string `"700"`
 
 ```ruby
-  # your answer here
+  007.to_s.reverse
 ```
 
 Please write the code that takes the following array `[23,56,3,7]` sorts it
 then reverses that array returning and modifying the refrence to the original
 array.
 
 ```ruby
-  # your answer here
+  [23,56,3,7].sort!
```

Comment:
This is half of what you needed to do.  The directions also ask you to reverse it.